app/router.jsx

```diff
@@ -1,25 +1,48 @@
 import React from 'react'
 import {createHistory} from 'history'
-import {Router, Route, useRouterHistory} from 'react-router'
+import {Router, Route, useRouterHistory, IndexRoute} from 'react-router'
 import {render} from 'react-dom'
 import Application from './components/Application'
+import Login from './components/Login'
 import CreateJob from './components/CreateJob'
 import JobStatusList from './components/JobStatusList'
+import auth from './utils/auth.js'
 import Help from './components/Help'
 import About from './components/About'
 
+function redirectToLogin(nextState, replace) {
```

Comment:
Can you move this internal function to after the exported function?

One convention I'm trying to keep is to keep all of the exports (i.e., public interface) of a module at the top of the file ([reference](https://github.com/venicegeo/bf-ui/blob/master/app/api/algorithms.js#L3-L6)).